raiden/tests/utils/smoketest.py

```diff
@@ -511,7 +512,7 @@ def setup_smoketest(
     print_step: StepPrinter,
     free_port_generator: Iterator[Port],
     debug: bool = False,
-    stdout: IO = None,
```

Comment:
Why is this necessary?

Reply:
Sorry, I now remember the section in the raiden dev-guide about default-`None` typing.